frontend/src/components/ControlPanel/ControlPanel.tsx

```diff
@@ -57,6 +57,7 @@ function ControlPanel({
 			{(currentLevel === LEVEL_NAMES.LEVEL_3 ||
 				currentLevel === LEVEL_NAMES.SANDBOX) && (
 				<>
+					<h2 className="visually-hidden">Chatbot Configurations</h2>
```

Comment:
unsure about "Chatbot Configurations". I'd argue the defence configurations are not chatbot configurations. So I guess I'd recommend either just "configurations" or "defence and model configurations"

Reply:
if we don't want to lump the view documents button with the config, can we give it its own header?

Reply:
I don't like the idea of having a separate header for the document viewer button, the idea of headers is to create skip links. While the configurations don't change the chatbot itself, they do change how the chatbot interacts in practice with the user. 
How about "Chatbot System Configurations"?